tests/test_cpu.py

```diff
@@ -102,6 +102,39 @@ def _open(not_used_arg):
 
     return archspec.cpu.TARGETS[target]
 
+@pytest.fixture(
+    params=[
+        "linux-centos7-thunderx2",
+        "linux-amazon-cortex_a72",
+        "linux-amazon-neoverse_n1",
+        "linux-amazon-neoverse_v1",
+        "linux-ubuntu22.04-neoverse_v2",
+        "linux-rhel9-neoverse_v2",
+        "linux-rhel9-neoverse_n2",
+        "linux-rhel8-neoverse_v1",
+    ]
+)
+def expected_linux_target(request, monkeypatch):
```

Comment:
There is no need to add a cut and paste fixture, which mimics `expected target`. Let's just add new tests to `expected_targets` and ensure everything is still detected correctly.

Reply:
Hi @alalazo  I have addressed this point but I'll move over to https://github.com/archspec/archspec/pull/177 and push it there.